frontend/src/components/Recommendation/RestaurantElement.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Container from '@mui/material/Container';
+import ListItemText from '@mui/material/ListItemText';
+import ListItemButton from '@mui/material/ListItemButton';
+import Collapse from '@mui/material/Collapse';
+import ExpandMore from '@mui/icons-material/ExpandMore';
+import ExpandLess from '@mui/icons-material/ExpandLess';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import Checkbox from '@mui/material/Checkbox';
+import ListItem from '@mui/material/ListItem';
+import RestaurantIcon from '@mui/icons-material/Restaurant';
+
+const RestaurantElement = ({ restaurant }) => {
```

Comment:
if you use `<APIProvider />` to handle calling apis and their responses, you can do:

`const {data, error} = useAPI();` here instead of passing in the prop from parent component.
(you might get into prop passing hell)